### PR TITLE
bring the documentation inline with the EAP slides

### PR DESCRIPTION
**Pull request by yveskerwyn**

### CODE REVIEW

File: docs/setup/setup.md [hunk 1]
```diff
@@ -38,10 +56,10 @@ cd /tmp
 export BRANCH="1.1.0-alpha-4"
 export ZEROTIERNWID="<Your ZeroTier network ID>"
 export ZEROTIERTOKEN="<Your ZeroTier token>"
-export ITSYOUONLINEORG="<itsyou.online organization>"
+export ITSYOUONLINEORG="<ItsYou.online organization>"
 export DOMAIN="<Your domain name>"
 curl -o install-orchestrator.sh https://raw.githubusercontent.com/zero-os/0-orchestrator/${BRANCH}/scripts/install-orchestrator.sh
-bash install-orchestrator.sh $BRANCH $ZEROTIERNWID $ZEROTIERTOKEN <$ITSYOUONLINEORG> [<$DOMAIN> [--development]]
+bash install-orchestrator.sh $BRANCH $ZEROTIERNWID $ZEROTIERTOKEN <$ITSYOUONLINEORG> \[<$DOMAIN> \[--development]]
```
Comment (zaibon) [inline docs/setup/setup.md]: Why the backslash here ?

Comment (yveskerwyn) [inline docs/setup/setup.md]: because Atom has problems formatting anything that comes at [

Comment (FastGeert) [inline docs/setup/setup.md]: You mean the markdown preview in atom ?

Comment (yveskerwyn) [inline docs/setup/setup.md]: yes

Comment (zaibon) [inline docs/setup/setup.md]: OK but then this has nothing to do with the actual code. Uta just an atom bug. That should not be reflected in the doc

**Pull request merged**